src/chain_spec.rs

```diff
@@ -267,6 +267,7 @@ pub fn testnet_genesis(
                 2_365_296_803_653,
             )],
         }),
+		pallet_vesting: Some(Default::default()),
```

Comment:
@vedhavyas Do you know if I need to modify the `*-raw-spec.json` files to add the empty definition of the `pallet_vesting` like above?